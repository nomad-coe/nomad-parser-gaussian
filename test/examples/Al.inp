%mem=24000000
#B1B95/6-31G** 

atom calc
casno=7429905
method=55
basis=3

0 2
Al

