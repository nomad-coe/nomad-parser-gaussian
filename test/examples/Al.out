 Entering Gaussian System, Link 0=g09
 Initial command:
 /usr/local/gaussian/g09.b01.em64t.linda/g09/l1.exe /tmp/rdj3/g09-29284/Gau-18735.inp -scrdir=/tmp/rdj3/g09-29284/
 Entering Link 1 = /usr/local/gaussian/g09.b01.em64t.linda/g09/l1.exe PID=     18736.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2010,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision B.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2010.
 
 ******************************************
 Gaussian 09:  EM64L-G09RevB.01 12-Aug-2010
                 9-Mar-2012 
 ******************************************
 %NProc=8
 Will use up to    8 processors via shared memory.
 %mem=424000000
 ---------------------------------------------------------
 #B3LYP/aug-cc-pVTZ scf=verytight integral(grid=ultrafine)
 ---------------------------------------------------------
 1/38=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=16,6=1,7=10,11=2,16=1,25=1,30=1,74=-5,75=-5/1,2,3;
 4//1;
 5/5=2,17=3,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,9=1/99;
 ----------------------------------------------------------------------
 al- Aluminum atom anion casno=7429905 state=1 config=1 method=2 basis=
 20
 ----------------------------------------------------------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 AL                    0.        0.        0. 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1         13           0        0.000000    0.000000    0.000000
 ---------------------------------------------------------------------
 Stoichiometry    Al(1-)
 Framework group  OH[O(Al)]
 Deg. of freedom     0
 Full point group                 OH      NOp  48
 Largest Abelian subgroup         D2H     NOp   8
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1         13           0        0.000000    0.000000    0.000000
 ---------------------------------------------------------------------
 Standard basis: Aug-CC-pVTZ (5D, 7F)
 There are    12 symmetry adapted basis functions of AG  symmetry.
 There are     3 symmetry adapted basis functions of B1G symmetry.
 There are     3 symmetry adapted basis functions of B2G symmetry.
 There are     3 symmetry adapted basis functions of B3G symmetry.
 There are     2 symmetry adapted basis functions of AU  symmetry.
 There are     9 symmetry adapted basis functions of B1U symmetry.
 There are     9 symmetry adapted basis functions of B2U symmetry.
 There are     9 symmetry adapted basis functions of B3U symmetry.
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
    50 basis functions,   115 primitive gaussians,    59 cartesian basis functions
     7 alpha electrons        7 beta electrons
       nuclear repulsion energy         0.0000000000 Hartrees.
 NAtoms=    1 NActive=    1 NUniq=    1 SFac= 1.00D+00 NAtFMM=   50 NAOKFM=F Big=F
 One-electron integrals computed using PRISM.
 NBasis=    50 RedAO= T  NBF=    12     3     3     3     2     9     9     9
 NBsUse=    50 1.00D-06 NBFU=    12     3     3     3     2     9     9     9
 Harris functional with IExCor=  402 diagonalized for initial guess.
 ExpMin= 1.46D-02 ExpMax= 2.06D+05 ExpMxC= 1.99D+03 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1
 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         Omega=  0.000000  0.000000  1.000000  0.000000  0.000000 ICntrl=     500 IOpCl=  0
         NMat0=    1 NMatS0=    1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
         I1Cent=           4 NGrid=           0.
 Petite list used in FoFCou.
 Initial guess orbital symmetries:
       Occupied  (A1G) (A1G) (T1U) (T1U) (T1U) (A1G) (T1U)
       Virtual   (T1U) (T1U) (A1G) (T1U) (T1U) (T1U) (T2G) (T2G)
                 (T2G) (EG) (EG) (T2G) (T2G) (T2G) (EG) (EG) (T1U)
                 (T1U) (T1U) (A1G) (T2U) (T2U) (T2U) (A2U) (T1U)
                 (T1U) (T1U) (T2G) (T2G) (T2G) (EG) (EG) (T2U)
                 (T2U) (T2U) (A2U) (T1U) (T1U) (T1U) (A1G) (T1U)
                 (T1U) (T1U)
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 Keep R1 ints in memory in canonical form, NReq=3559651.
 Density matrix breaks symmetry, PCut= 1.00D-04
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-04
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-04
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-04
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-04
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-04
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-04
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-04
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-04
 Density has only Abelian symmetry.
 Density matrix breaks symmetry, PCut= 1.00D-04
 Density has only Abelian symmetry.
 SCF Done:  E(RB3LYP) =  -242.389700912     A.U. after    9 cycles
             Convg  =    0.7910D-09             -V/T =  2.0034

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A1G) (A1G) (T1U) (T1U) (T1U) (?A) (?B)
       Virtual   (?B) (?B) (?A) (?B) (?B) (?B) (?C) (?C) (?C) (?A)
                 (?A) (?C) (?A) (?C) (?C) (?A) (?B) (?B) (?B) (?A)
                 (?B) (A2U) (?B) (?B) (?B) (?B) (?B) (T2G) (?A)
                 (T2G) (T2G) (?A) (?B) (?B) (?B) (A2U) (?B) (?B)
                 (?B) (A1G) (T1U) (T1U) (T1U)
 Unable to determine electronic state:  an orbital has unidentified symmetry.
 Alpha  occ. eigenvalues --  -55.90858  -3.98439  -2.53890  -2.53890  -2.53183
 Alpha  occ. eigenvalues --   -0.14707   0.04154
 Alpha virt. eigenvalues --    0.06873   0.06873   0.11144   0.11276   0.11649
 Alpha virt. eigenvalues --    0.11649   0.16553   0.16553   0.17309   0.17313
 Alpha virt. eigenvalues --    0.17442   0.36907   0.36908   0.37273   0.37273
 Alpha virt. eigenvalues --    0.37387   0.37412   0.37412   0.37996   0.38088
 Alpha virt. eigenvalues --    0.39755   0.39755   0.39994   0.39994   0.40534
 Alpha virt. eigenvalues --    0.40534   0.40628   0.91783   0.91783   0.92696
 Alpha virt. eigenvalues --    0.92696   0.93384   1.04866   1.04866   1.05411
 Alpha virt. eigenvalues --    1.05411   1.06226   1.06226   1.06431   1.75528
 Alpha virt. eigenvalues --    2.11663   2.11663   2.13025
          Condensed to atoms (all electrons):
              1
     1  Al  14.000000
 Mulliken atomic charges:
              1
     1  Al  -1.000000
 Sum of Mulliken atomic charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
              1
     1  Al  -1.000000
 Sum of Mulliken charges with hydrogens summed into heavy atoms =  -1.00000
 Electronic spatial extent (au):  <R**2>=             89.8727
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=              0.0000  Tot=              0.0000
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -28.0764   YY=            -28.0764   ZZ=            -64.7292
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             12.2176   YY=             12.2176   ZZ=            -24.4352
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              0.0000  ZZZ=              0.0000  XYY=              0.0000
  XXY=              0.0000  XXZ=              0.0000  XZZ=              0.0000  YZZ=              0.0000
  YYZ=              0.0000  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=           -210.4685 YYYY=           -210.4685 ZZZZ=           -964.4363 XXXY=              0.0000
 XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=            -70.1562 XXZZ=           -196.7924 YYZZ=           -196.7924
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
 N-N= 0.000000000000D+00 E-N=-5.803947170106D+02  KE= 2.415679408169D+02
 Symmetry AG   KE= 1.819256577668D+02
 Symmetry B1G  KE= 0.000000000000D+00
 Symmetry B2G  KE= 0.000000000000D+00
 Symmetry B3G  KE= 0.000000000000D+00
 Symmetry AU   KE= 0.000000000000D+00
 Symmetry B1U  KE= 2.039512239737D+01
 Symmetry B2U  KE= 1.962358032638D+01
 Symmetry B3U  KE= 1.962358032638D+01
 1\1\GINC-N233\SP\RB3LYP\Aug-CC-pVTZ\Al1(1-)\RDJ3\09-Mar-2012\0\\#B3LYP
 /aug-cc-pVTZ scf=verytight integral(grid=ultrafine)\\al- Aluminum atom
  anion casno=7429905 state=1 config=1 method=2 basis=20\\-1,1\Al,0,0.,
 0.,0.\\Version=EM64L-G09RevB.01\HF=-242.3897009\RMSD=7.910e-10\Dipole=
 0.,0.,0.\Quadrupole=-18.1669729,9.0834864,9.0834864,0.,0.,0.\PG=OH [O(
 Al1)]\\@


 LAWS OF PROGRAMMING DEFINITION:  A WORKING PROGRAM
                                  IS ONE THAT HAS
                                  ONLY UNOBSERVED
                                  BUGS.
 Job cpu time:  0 days  0 hours  0 minutes 18.6 seconds.
 File lengths (MBytes):  RWF=      5 Int=      0 D2E=      0 Chk=      1 Scr=      1
 Normal termination of Gaussian 09 at Fri Mar  9 14:47:52 2012.

